qa/tasks/vstart_runner.py

```diff
@@ -560,12 +560,14 @@ def run_as_root(self, args, wait=True, stdin=None, check_status=True):
                                       check_status=check_status,
                                       omit_sudo=False)
 
-    def testcmd(self, args, wait=True, stdin=None, omit_sudo=False):
+    def testcmd(self, args, wait=True, stdin=None):
+        """
+        Conduct a positive test for the given command.
+        """
         # FIXME maybe should add a pwd arg to teuthology.orchestra so that
         # the "cd foo && bar" shenanigans isn't needed to begin with and
         # then we wouldn't have to special case this
-        return self.run_shell(args, wait=wait, stdin=stdin, check_status=False,
-                              omit_sudo=omit_sudo)
+        return self.run_shell(args=args, wait=wait, stdin=stdin)
```

Comment:
I've moved this commit to a separate PR and the PR got merged recently. I am getting rid of `testcmd()` since it's same as `run_shell()` from behaviour's POV (both set the `check_status`'s default value to `True`).